docs/src/getting-started/installation.md

```diff
@@ -46,7 +46,7 @@ sudo apt-get update && sudo apt-get upgrade && sudo apt-get install -y pkg-confi
 To install the JavaScript package.
 
 ```bash
-npm install -g @project-serum/anchor
+npm install @project-serum/anchor
```

Comment:
Need to be mentioned that works only on linux/x86_64 :disappointed: 

The next section should be corrected (`Make sure your `NODE_PATH` is set properly so that globally installed modules
can be resolved.`).

Reply:
I think you're referring to `@project-serum/anchor-cli`?

Reply:
Oh, indeed. Thanks for fix :+1: